spec/features/users_can_view_a_voluntary_conversion_task_list_spec.rb

```diff
@@ -1,19 +0,0 @@
-require "rails_helper"
```

Comment:
This is a voluntary task list spec - did you mean to remove it?

Reply:
Nope! Thank you!